test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -85,3 +109,15 @@ bytes SolidityExecutionFramework::compileContract(
 		cout << "metadata: " << m_compiler.metadata(contractName) << endl;
 	return obj.bytecode;
 }
+
+void SolidityExecutionFramework::addPreamble(string& _sourceCode)
+{
+	// Silence compiler version warning
+	string preamble = "pragma solidity >=0.0;\n";
+	if (
+		solidity::test::CommonOptions::get().useABIEncoderV2 &&
+		_sourceCode.find("pragma experimental ABIEncoderV2;") == string::npos
+	)
+		preamble += "pragma experimental ABIEncoderV2;\n";
+	_sourceCode = preamble + _sourceCode;
```

Comment:
I think that it needs an explicit `move`:
```suggestion
	_sourceCode = preamble + move(_sourceCode);
```